.eslintrc.js

```diff
@@ -15,8 +15,8 @@ module.exports = {
     "mozilla"
   ],
   "rules": {
-    "no-console": "off", // TODO: warn
+    "eqeqeq": "warn",
```

Comment:
I spotted some `==` in a different PR. Not sure of our collective thoughts on `===`/`!==` (versus their less strict `==`/`!=` counterparts), but this rule currently triggers 4 warnings:

```
/Users/pdehaan/dev/github/mozilla/tab-split-2/addon/background.js
  69:20  warning  Expected '===' and instead saw '=='  eqeqeq
  71:27  warning  Expected '===' and instead saw '=='  eqeqeq

/Users/pdehaan/dev/github/mozilla/tab-split-2/addon/sidebar.js
  2:60  warning  Expected '!==' and instead saw '!='  eqeqeq
  6:20  warning  Expected '===' and instead saw '=='  eqeqeq
```

Reply:
I think we'll want to just change these, I opened #45 about it.